pkg/framework/deployment.go

```diff
@@ -66,6 +78,27 @@ func IsDeploymentAvailable(c client.Client, name, namespace string) bool {
 	return true
 }
 
+// IsDeploymentSynced returns true if provided deployment spec matched one found on cluster
+func IsDeploymentSynced(c client.Client, dep *kappsapi.Deployment, name, namespace string) bool {
+	if err := wait.PollImmediate(1*time.Second, WaitLong, func() (bool, error) {
+		d, err := GetDeployment(c, name, namespace)
+		if err != nil {
+			glog.Errorf("Error getting deployment: %v", err)
+			return false, nil
+		}
+		if !reflect.DeepEqual(d.Spec, dep.Spec) {
+			return true, fmt.Errorf("Deployment %q is updated. Spec is not equal to: %v",
```

Comment:
It's not updated right?
```suggestion
			return true, fmt.Errorf("Deployment %q is not updated. Spec is not equal to: %v",
```